Classes/Domain/Model/Component.php

```diff
@@ -117,6 +111,9 @@ public function getFixtures(): array
                 $loader = GeneralUtility::makeInstance(YamlFileLoader::class);
                 $fixtures = $loader->load($fixtureFile) ?? [];
                 break;
+            case 'php':
+                $fixtures = require $fixtureFile;
```

Comment:
Maybe we should validate that there is an array coming? Also the other cases take care of failures and set to empty array. 

I think both could achieved with this array cast? 
```suggestion
                $fixtures = (array) require $fixtureFile;
```